pkg/smartcontract/parameter.go

```diff
@@ -25,7 +25,7 @@ type Parameter struct {
 	// Type of the parameter.
 	Type ParamType `json:"type"`
 	// The actual value of the parameter.
-	Value interface{} `json:"value"`
+	Value interface{} `json:"value,omitempty"`
```

Comment:
Do we need `json` here at all? We're converting to/from JSON via `rawParameter` anyway.